learning_assistant/__init__.py

```diff
@@ -2,6 +2,6 @@
 Plugin for a learning assistant backend, intended for use within edx-platform.
 """
 
-__version__ = '4.4.4'
+__version__ = '4.4.5'
```

Comment:
Can you include a changelog entry?

Reply:
There is one already: https://github.com/edx/learning-assistant/pull/129/files#diff-2c623f3c6a917be56c59d43279244996836262cb1e12d9d0786c9c49eef6b43c